apps/server/src/modules/learnroom/service/common-cartridge-export.service.ts

```diff
@@ -23,6 +23,8 @@ export class CommonCartridgeExportService {
 		const builder = new CommonCartridgeFileBuilder({
 			identifier: `i${course.id}`,
 			title: course.name,
+			copyrightOwners: this.mapCourseTeachersToCopyrightOwners(course),
+			currentYear: new Date().getFullYear().toString(),
```

Comment:
Should we add the current year or the creation date of the course here?

Reply:
The common practice I have seen so far, is that in most cases the creation date of the resource itself is used for the copyright. In this case, I tend to the creation date of the course, because the export is a copy of the course and not something new.